api-reference/docstubs/api/microsoft.fast.pi.viewgenerator.v2.profile.models-profile-update.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Update profile"
```

Comment:
I'm not sure this is possible.  the only property on profile is `id` and I think it's system generated...

Reply:
Looks like the profile API doesn't expose the topic for updating a user's profile; it's probably not supported.

Yes, in the auto-generation workflow, is it that the individual authors will continue to select those CRUD topics that they actually support?

Reply:
Yes - afraid so, but I wonder in this case whether we could have actually been smarter.  Probably a P3 though.

BUG BUG